### PR TITLE
have the functions working. form is in new component.

### PR DESCRIPTION
**Pull request by SimoneOdegard**

### CODE REVIEW

**Review by JessiVelazquez — APPROVED** [review]
Looks good

File: src/Display.js [hunk 1]
```diff
@@ -17,18 +17,29 @@ class Display extends React.Component {
   }
 
   componentDidMount = async () => {
+    this.getMovies();
+  }
+
+  getMovies = async () => {
+    console.log('getting movies');
     const getMovies = await axios.post(`${API_SERVER}/user`, {email: this.props.auth0.user.email})
```
Comment (ganttArt) [inline src/Display.js:25]: this is going to need to be a get request right?

Comment (ganttArt) [reply src/Display.js:25]: I think this route needs to be changed to /movies too, it wont be /user

Comment (SimoneOdegard) [reply src/Display.js]: updated to .get

Comment (SimoneOdegard) [reply src/Display.js]: updated route to /movies

**Review by ganttArt — APPROVED** [review]

**Pull request merged**